go-controller/pkg/ovn/controller/egressservice/egressservice_zone.go

```diff
@@ -538,6 +541,11 @@ func (c *Controller) repair() error {
 		c.services[key].v6RemoteEndpoints.Insert(v6ToAdd...)
 	}
 
+	// re-set the global egressservice address set
+	if err := c.setPodIPsInAddressSet(createIPAddressNetSlice(allLocalV4Eps, allLocalV6Eps)); err != nil {
```

Comment:
Customary question: nobody else might be wirting to this AS at the same time, right?

I was looking at https://github.com/ovn-org/ovn-kubernetes/blob/017427db418a8491edfd2088de04943881af14b7/go-controller/pkg/ovn/egressip.go#L1324-L1325
but I guess that's not concurrent.

Reply:
> Customary question: nobody else might be wirting to this AS at the same time, right?

Yup, only the egressService controller should be modifying that address set.
Currently the whole egressService controller is being locked on reconciliation.